lib/segment/src/index/hnsw_index/hnsw.rs

```diff
@@ -191,7 +194,10 @@ impl<TGraphLinks: GraphLinks> HNSWIndex<TGraphLinks> {
         let ef = max(req_ef, top);
 
         let vector_storage = self.vector_storage.borrow();
-        let ignore_quantization = params.map(|p| p.ignore_quantization).unwrap_or(false);
+        let ignore_quantization = params
+            .and_then(|p| p.quantization)
+            .map(|q| q.ignore)
+            .unwrap_or(default_quantization_ignore_value());
```

Comment:
Thanks for using `default_quantization_ignore_value`, that is better than magic value `false`

Reply:
:heavy_check_mark: